kahuna/public/js/components/gu-date-range/gu-date-range.js

```diff
@@ -53,126 +53,125 @@ guDateRange.directive('guDateRange', [function () {
-                maxDate: tenYearsFromNow,
-                yearRange: 100,
-                firstDay: parseInt(ctrl.guFirstDay),
-                format: iso8601Format
-            });
-
-
-            var pikaEnd = new Pikaday({
-                field: endInput,
-                container: endContainer,
-                bound: false,
-                maxDate: tenYearsFromNow,
-                firstDay: parseInt(ctrl.guFirstDay),
-                format: iso8601Format,
-                yearRange: 100
-            });
-
-            $scope.$watch('pikaStartValue', function (pikaStartValue) {
-                var date = (pikaStartValue && new Date(pikaStartValue)) || new Date();
-                pikaEnd.setMinDate(date);
-                pikaEnd.hide();
-                pikaEnd.show();
-            });
-
-            $scope.$watch('pikaEndValue', function () {
-                pikaStart.hide();
-                pikaStart.show();
-            });
-
-            $scope.$watch('ctrl.guEndDate',   resetView);
-
-            $scope.$on('$destroy', function() {
-                pikaStart.destroy();
-                pikaEnd.destroy();
-            });
-
-            function resetView() {
-                ctrl.setDateRangeForDisplay();
-
-                pikaStart.setDate();
-                pikaEnd.setDate();
-
-                pikaStart.setDate(ctrl.guStartDate);
-                pikaEnd.setDate(ctrl.guEndDate);
-
-                ctrl.closeOverlay();
-            }
-
-            function getStartValue () {
-                var start = pikaStart.getDate();
-                return start ? moment(start).startOf('day').toDate() : undefined;
-            }
-
-            function getEndValue () {
-                var end = pikaEnd.getDate();
-                return end ? moment(end).endOf('day').toDate() : undefined;
-            }
-
-            function setPresetDate (preset) {
-                ctrl.save(preset);
-                resetView();
-            }
-
-            function setCustomRange () {
-                var start = getStartValue();
-                var end = getEndValue();
-
-                ctrl.save(start, end);
-                resetView();
-            }
-
-            function clearStart () {
-                pikaStart.setDate();
-                pikaStart.gotoToday();
-            }
-
-            function clearEnd () {
-                pikaEnd.setDate();
-                pikaEnd.gotoToday();
-            }
-
-            $scope.cancel = resetView;
-            $scope.save = setCustomRange;
-            $scope.setPresetDate = setPresetDate;
-            $scope.clearStart = clearStart;
-            $scope.clearEnd = clearEnd;
-
-            resetView();
+          ctrl.guDateFormat = ctrl.guDateFormat || 'DD-MMM-YYYY';
+          ctrl.guAnyTimeText = ctrl.guAnyTimeText || 'anytime';
+          ctrl.guFirstDay = ctrl.guFirstDay || 0;
+          var startInput = el[0].querySelectorAll('.gu-date-range__input__start--hidden')[0];
```

Comment:
minor: worth creating a variable for `el[0]` for DRYness?

Reply:
Yep, sounds good to me. It's a pattern used in the main.js file too.